### ISSUE TITLE
min command/motor idle throttle value changeable

### ISSUE THREAD

**Issue by hsuq510** [codecae/betaflight-tx-lua-revised#15]
A nice feature to have would be to change the min command or the motor idle throttle via script as this could be very helpfull to optimize setup on the fly.

**Comment by PrestonHack** [comment]
Min throttle is in there

**Comment by codecae** [comment]
That mid is for expo.  would need to build a different screen for these

**Comment by codecae** [comment]
https://github.com/betaflight/betaflight/blob/master/src/main/fc/fc_msp.c#L922

**Comment by PrestonHack** [comment]
oh miss read

**Comment by codecae** [comment]
Easy to confuse the two.. no biggie

**Comment by codecae** [comment]
Added an RX screen.  Let me know if these features are what you are asking for.  I believe the idle is already on the PWM screen

**Comment by codecae** [comment]
going once?

**Comment by hsuq510** [comment]
yes installed but not tested yet, very usefull ! the idle offset seems to go from 4.5 (default to 20) and its not possible to get under 20. Other value seems fine.

**Comment by codecae** [comment]
can you provide a screenshot of field you're expecting to update in the configurator?

**Comment by hsuq510** [comment]
![image](https://user-images.githubusercontent.com/1420296/30875407-b95c5498-a2f3-11e7-87a0-9a1364dcb889.png)

**Comment by hsuq510** [comment]
The idle offset goes 20 and above, never get back to < 20

**Comment by PrestonHack** [comment]
what if you change the protocol then change it back? I will take a look at this on my rig

**Comment by PrestonHack** [comment]
Hey null, not sure of the line number but this: min=2000,max=32000,to=SMLSIZE,scale=100}}
this is the last field value and the range appears to be the issue.

**Comment by PrestonHack** [comment]
shouldn't this be something like min=0, max=10000,to=SMLSIZE,scale=100}}

**Comment by codecae** [comment]
Updated. https://github.com/codecae/betaflight-tx-lua-revised/pull/16

**Comment by codecae** [comment]
At your earliest convenience, please verify that this issue has been resolved to your expectations... thanks!

**Comment by PrestonHack** [comment]
will do

**Comment by hObBiT-iM-bLuTrAuScH** [comment]
Thumbs up. I love this project since my usb port broke. Looking for an update to calibrate all the sensors and edit modi. How about implementing backups? Could it be possible to implement  versionupgrade or cli in the script?

**Comment by codecae** [comment]
@hObBiT-iM-bLuTrAuScH  Can you please track your enhancement request in a separate issue?  If I close this one because the original issue is resolved, your idea will become lost.  Thanks!

**Comment by hObBiT-iM-bLuTrAuScH** [comment]
I opened a new issue on modes and sensor calibration, I´m not sure if it is possible to implement cli, backup or versionupgrade. I´ll stay tuned and maybe we can check this later on. What you think?

**Comment by codecae** [comment]
I don't really see how CLI would even work.. You don't have a keyboard on your taranis.  No way to do firmware upgrades as you can't access the FC while it's in bootloader mode.

**Comment by hObBiT-iM-bLuTrAuScH** [comment]
That´s the reason I did not open a issue on that yet. You could scroll through the Alphabet with +/- and select with enter, but it would be a pain in the ass. On the other hand you could access EVERY switch without betaflight configurator. For there is one setting that will not be covered by this superscript and there is no other possibillity to enter cli, that would be a nice lifesafer.

See the problem with the bootloader. Your script let´s me dream to much. Maybe I thought it might be magic.

**Comment by codecae** [comment]
Generally most of the things you would need access to can be found using MSP, which is what this script already uses.
